src/components/illustration/illustration.stories.tsx

```diff
@@ -1,24 +1,24 @@
-import React, { CSSProperties } from 'react';
 import { ComponentStory } from '@storybook/react';
+import * as Illustrations from '@teamleader/ui-illustrations';
+import React from 'react';
 import { addStoryInGroup, LOW_LEVEL_BLOCKS } from '../../../.storybook/utils';
 import Box from '../box';
-import description from './description.md';
-import * as Illustrations from '@teamleader/ui-illustrations';
 import { TextSmall } from '../typography';
+import description from './description.md';
 
-const gridStyles: CSSProperties = {
+const gridStyles: Record<string, any> = {
```

Comment:
because our `Box` components doesn't accepts all CSSProperties in its full glory. for example:
`alignContent: 'center' | 'flex-start' | 'flex-end' | 'space-around' | 'space-between' | 'space-evenly';`
this picking of properties is because then styling is done via classes:
`[theme[`align-content-${alignContent}`]]: alignContent,`